app/src/main/res/values-nl/styles.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <style name="AppTheme" parent="@style/Theme.Leanback" />
+
+    <style name="TextCardStyle" parent="Widget.Leanback.BaseCardViewStyle">
+        <item name="cardBackground">@null</item>
```

Comment:
Hmmm this whole file can be removed

Reply:
Didn't know styles.xml was not needed for translations. First time hacking an Android app... ;)